modules/learn/pages/clusters-and-availability/groups.adoc

```diff
@@ -15,8 +15,32 @@ Specifically, it protects a cluster from large-scale infrastructure failure, thr
 Each group is created by an appropriately authorized administrator, and specified to contain a subset of the nodes within a Couchbase Cluster.
 Following group-definition and rebalance, the active vBuckets for any defined bucket are located on one group, while the corresponding replicas are located on another group: indexes and _index replicas_, maintained by the Index Service, are distributed across groups in a similar way.
 
-A server group can be automatically _failed over_: thus, if the entire group goes offline, and active vBuckets and indexes are thereby inaccessible, the replica vBuckets and replica indexes that remain available on another group can be automatically promoted to active status.
-In 7.1+, this is achieved by setting the maximum count for auto-failover to a value equal to or greater than the number of nodes in the server group.
+A server group can be automatically _failed over_: thus, if the entire group goes offline, and active vBuckets and indexes are thereby inaccessible, then the replica vBuckets and replica indexes that remain available on another group can be automatically promoted to active status.
+
+NOTE: For the vBuckets and replica indexes to be automatically promoted to active, the conditions specified in xref:./automatic-failover.adoc#auto-failover-constraints[Auto-failover Constraints] must apply.
+
+See xref:learn:clusters-and-availability/automatic-failover.adoc[Automatic Failover].
+
+Note that In 7.1+, automatic failover can fail over more than three nodes concurrently: this has permitted the removal of pre-7.1 interfaces that were specific to triggering auto-failover for server groups.
+Consequently, for auto-failover of a server group to be possible, the maximum count for auto-failover must be established by the administrator as a value equal to or greater than the number of nodes in the server group.
+
+[IMPORTANT]
+====
+For auto-failover to occur,
+all of the auto-failover constraints
+listed in xref:./automatic-failover.adoc#auto-failover-constraints[Auto-failover Constraints] must be met,
+including the majority quorum requirement:
+i.e., the remaining nodes must be able to form a majority quorum to be able to initiate an auto-failover. +
+If, for example, you have two server groups, with equal number of nodes in each,
+and if all the nodes in one of the server groups fail at the same time,
+even if you have the maximum count for auto-failover
+set to a value equal to or greater than the number of nodes in the server group that failed,
+auto-failover cannot occur since the remaining nodes in the remaining server group cannot form a majority quorum.
+====
+
+
+Note also that in pre-7.1, interfaces for triggering auto-failover for server groups have been _removed_ from 7.1: therefore, programs that attempt to use the pre-7.1 interfaces with 7.1+ will _fail_. +
```

Comment:
Seems redundant given the info is (mostly) in line 24.  Maybe the text following the colon ("therefore...") should be moved to line 24 if needed.  If there's a reason to keep line 42, re-word the awkward beginning "Note also that in pre-7.1, interfaces..." to something like "Note also that pre-7.1 interfaces..."

Reply:
I've moved the text closer to the note about 7.1.